token/core/logging/logger.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package logging
+
+import (
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/services/flogging"
+	"go.uber.org/zap/zapcore"
+)
+
+// Logger provides logging API
+type Logger interface {
+	Debug(args ...interface{})
+	Debugf(format string, args ...interface{})
+	Error(args ...interface{})
+	Errorf(format string, args ...interface{})
+	Fatal(args ...interface{})
+	Fatalf(format string, args ...interface{})
+	Info(args ...interface{})
+	Infof(format string, args ...interface{})
+	Panic(args ...interface{})
+	Panicf(format string, args ...interface{})
+	Warn(args ...interface{})
+	Warnf(format string, args ...interface{})
+	IsEnabledFor(level zapcore.Level) bool
+}
+
+func DriverLogger(prefix string, networkID string, channel string, namespace string) Logger {
+	return flogging.MustGetLogger(prefix + extension(networkID) + extension(channel) + extension(namespace))
```

Comment:
We can also do it like this
```
func isEmptyString(s string) bool { return len(s) == 0 }

func loggerName(parts ...string) string {
 return strings.Join(slices.DeleteFunc(parts, isEmptyString), ".")}
```
but it's a matter of taste

Reply:
👍